web/concrete/src/User/User.php

```diff
@@ -125,10 +127,10 @@ public function getUserInfoObject()
 
     public function __construct()
     {
-        $app = Application::getFacadeApplication();
+        $this->app = Application::getFacadeApplication();
```

Comment:
I get what you mean about scope creep, but as long as we don't refactor that, this should be faster, shouldn't it? It consumes the same amount of memory and we simply have less calls to `Application::getFacadeApplication`? I guess it's late, will re-read this tomorrow..